internals/cli/masker/stream.go

```diff
@@ -0,0 +1,122 @@
+package masker
+
+import (
+	"bytes"
+	"io"
+	"sync"
+)
+
+// stream is a buffered io.Writer that masks all secrets written on it using a multipleMatcher.
```

Comment:
Do all streams use the same `multipleMatcher`? Is it relevant to mention the `multipleMatcher` in the docs here?

Reply:
Removed the reference to the matcher.